environment/src/main/java/com/sequenceiq/environment/environment/v1/converter/NetworkRequestToDtoConverter.java

```diff
@@ -110,4 +115,8 @@ private ServiceEndpointCreation getServiceEndpointCreation(EnvironmentNetworkReq
     private OutboundInternetTraffic getOutboundInternetTraffic(EnvironmentNetworkRequest network) {
         return Optional.ofNullable(network.getOutboundInternetTraffic()).orElse(OutboundInternetTraffic.ENABLED);
     }
+
+    private Boolean getUsePublicEndpointAccessGateway(EnvironmentNetworkRequest network) {
```

Comment:
use enum please

Reply:
I've created an enum `PublicEndpointAccessGateway` with values `ENABLED` and `DISABLED`, and used it in all the objects that have `usePublicEndpointAccessGateway`.